solutions/usaco-1011.mdx

```diff
@@ -95,4 +95,58 @@ public class triangles{
 
 </JavaSection>
 
+<CPPSection>
+
+```CPP
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	freopen("triangles.in", "r", stdin);
+	freopen("triangles.out", "w", stdout);
+	int n; cin >> n;
```

Comment:
```suggestion
	freopen("triangles.out", "w", stdout);
	
	int n; cin >> n;
```

Reply:
don't think this whitespace is needed tbh

Reply:
yea... what is the point of it